pkg/util/log/redact_test.go

```diff
@@ -154,14 +154,14 @@ func TestRedactedDecodeFile(t *testing.T) {
 			Infof(context.Background(), "marker: this is safe, stray marks ‹›, %s", "this is not safe")
 
 			// Retrieve the log writer and log location for this test.
-			info, ok := debugLog.getFileSink().mu.file.(*syncBuffer)
+			debugSink := debugLog.getFileSink()
+			info, ok := debugSink.mu.file.(*syncBuffer)
```

Comment:
Without knowing the internals, I'm not sure why this is safe, is there a reason you can't grab the lock here?

Reply:
Grab which lock?

Reply:
I see `debugSink.mu`, so I would've expected a `debugSink.mu.Lock()`before accessing `debugSink.mu.file`.